memcached/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM debian:jessie
```

Comment:
Let's move the memcached image, Dockerfile, etc to https://github.com/autopilotpattern/memcached/tree/wip so it's more easily reused in other projects.